Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -186,15 +189,25 @@ If not installing with Helm, here are the steps to install with manifests.
    changing the current context, you can specify the cluster name by setting
    the KUBECONTEXT environment variable.
 
-   ```console
-   export KUBECONTEXT=<cluster-name>
-   ```
+    ```console
+    export KUBECONTEXT=<cluster-name>
+    ```
+
+3. Here is the link for [import](https://github.com/rook/rook/blob/master/deploy/examples/external/import-external-cluster.sh) script. The script has used the `rook-ceph-external` namespace and few parameters that also have referenced from namespace variable. If user's external cluster has a different namespace, change the parameters in the script according to their external cluster settings. For example with `rook-ceph` namespace, these changes are needed on the following parameters in the script.
+
+    ```console
+    NAMESPACE=${NAMESPACE:="rook-ceph"}
```

Comment:
no need for this
```
 ROOK_EXTERNAL_CLUSTER_NAME=$NAMESPACE"-external"
    CLUSTER_ID_RBD=$NAMESPACE"-external"
    CLUSTER_ID_CEPHFS=$NAMESPACE"-external"
```
if we just change the namespace  others will be autoupdated

Reply:
Lets change the original import script to use `NAMESPACE=${NAMESPACE:="rook-ceph"}`

Reply:
I can change the original import script, but will keep some info if user have a different namespace. it was requested by @travisn